internal/config/main_config.go

```diff
@@ -98,7 +101,7 @@ var Sources = &config.RestEndpoints.Sources
 var Worker = &config.Worker
 
 func Initialize() {
-	err := viper.Unmarshal(&config)
+	err := parser.Viper.Unmarshal(&config)
```

Comment:
Using `viper` package (default singleton variable viper.v) is no longer possible due to bug in the library which needs v2 release, so I moved everything into a package where I create proper string replace configuration for it.